src/tls_migration.erl

```diff
@@ -0,0 +1,67 @@
+%% TLS migration script (temporary)
+%%
+%% Run from the console as:
+%% > l(tls_migration). tls_migration:main(ok).
+
+-module(tls_migration).
+
+-export([main/1]).
+
+-include_lib("ex_uri/include/ex_uri.hrl").
+-include("logplex_drain.hrl").
+-include("logplex_logging.hrl").
+
+main(_) ->
+    ets:foldl(fun(Drain, _Acc) ->
+                      case should_migrate_drain(Drain) of
+                          true ->
+                              migrate_drain_maybe(Drain);
+                          _ ->
+                              ok
+                      end
+              end, notused, drains),
```

Comment:
I would suggest passing in an empty list `[]` and using the accumulator. This way the failed drains can be returned to the caller and printed all at once instead of interleaved with everything else.